### ISSUE TITLE
Embargo Management Page link throws exception 

### ISSUE THREAD

**Issue by coblej** [samvera/hyrax#1419]
### Descriptive summary

Hyrax v1.0.3
On an embargoed work, clicking the "Embargo Management Page" link (which goes to, in an example case, http://localhost:3000/embargoes/nz805z68q/edit?locale=en) results in an error ... "uninitialized constant EmbargoesController".  Perhaps this is because the EmbargoesController is namespaced to Hyrax -- i.e., Hyrax::EmbargoesController?

Entries from `rake routes`:
```
               embargoes PATCH    /embargoes(.:format)                                  embargoes#update
                         GET      /embargoes(.:format)                                  embargoes#index
            edit_embargo GET      /embargoes/:id/edit(.:format)                         embargoes#edit
                 embargo DELETE   /embargoes/:id(.:format)                              embargoes#destroy
``` 

### Rationale

Appears to be a bug in the routes?

### Expected behavior

Correct controller is found.

### Actual behavior

"uninitialized constant EmbargoesController"

### Steps to reproduce the behavior

1. Set an embargo on a work.
2. Edit the work.
3. Click the "Embargo Management Page" link in the Visibility section.

### Related work

None that I know of.

**Comment by mjgiarlo** [comment]
@coblej The embargoes and leases controllers were not pulled over from CurationConcerns until relatively recently (late May), and currently live in `master`. See this PR: https://github.com/samvera/hyrax/pull/1042

You could look into backporting this to `1-0-stable`, but that may be a big, gnarly backport. Might you be open to using the latest Hyrax work?

**Comment by coblej** [comment]
@mjgiarlo I don't rule out using the v2.0.0.alpha code.  If I may, I'll contact you outside the context of this ticket about making that switch.

**Comment by mjgiarlo** [comment]
@coblej OK. Will write you back today.

**Comment by no-reply** [comment]
Closing this as fixed in 2.0. We'll not backport unless someone wants to reopen and pick it up.
